### PR TITLE
About page

### PR DESCRIPTION
**Pull request by meeragk99**

### CODE REVIEW

**Review by xchanti124 — COMMENTED** [review]
Looks good to me, however, I think you should wait for Olya's approval.

File: package.json
Comment (FlyingRatMan) [inline package.json]: We don't need this "express-body-parser-error-handler". Could you please delete it? `npm uninstall express-body-parser-error-handler`

File: package.json
Comment (FlyingRatMan) [inline package.json]: We don't need this "react-hook-form". Could you please delete it? `npm uninstall react-hook-form`

File: src/App.js
Comment (FlyingRatMan) [inline src/App.js]: Are we redirecting explicitly to `/404` anywhere? We already have a fallback for undefined routes.

File: src/components/About.js
Comment (FlyingRatMan) [inline src/components/About.js]: This should be removed.

File: src/components/About.js
Comment (FlyingRatMan) [inline src/components/About.js]: This should be removed.

**Review by FlyingRatMan — CHANGES_REQUESTED** [review]
There are things to be removed.

**Review by FlyingRatMan — APPROVED** [review]

**Pull request merged**
